src/tests/functional/plugin/shared/include/behavior/compiled_model/compiled_model_base.hpp

```diff
@@ -267,6 +269,47 @@ TEST_P(OVCompiledModelBaseTest, CanGetInputsInfo) {
     EXPECT_NO_THROW(auto inInfo = execNet.inputs());
 }
 
+TEST_P(OVCompiledModelBaseTest, CanGetProperty) {
```

Comment:
we have similar test: TEST_P(OVClassCompiledModelPropertiesTests, canCompileModelWithPropertiesAndCheckGetProperty)
in openvino/src/tests/functional/plugin/shared/src/behavior/compiled_model/properties.cpp